microprofile/rest-client/pom.xml

```diff
@@ -64,7 +64,6 @@
                     <plugin>
                         <groupId>fish.payara.maven.plugins</groupId>
                         <artifactId>payara-micro-maven-plugin</artifactId>
-                        <version>${payara.miro.maven.version}</version>
```

Comment:
While this is unrelated to the original issue, I considered this low risk enough to include here. This property is not set and the version is specified in the parent pom.

Reply:
On second thought, there's more than this issue in the rest client example. I've reverted the change.